### ISSUE TITLE
Comment for the first member of the structure is misplaced

### ISSUE THREAD

**Issue by op2786** [Vector35/binaryninja-api#6087]
**Version and Platform (required):**
- Binary Ninja Version: 4.2.6358-dev, 29712d38
- OS: macos
- OS Version: 15.1
- CPU Architecture: arm64

**Bug Description:**

<img width="474" src="https://github.com/user-attachments/assets/bde80090-d698-4cf2-a2f4-018746e300c1">

**Comment by xusheng6** [comment]
I feel strongly that it is a known issue, but I cannot find it at the moment

**Comment by op2786** [comment]
Yeah, I've that feeling, but I'm unable to find that issue too. Sorry if it is a duplicate.
